src/main/java/seedu/address/model/person/Appointment.java

```diff
@@ -0,0 +1,99 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+
+
+/**
+ * Represents an Appointment attached to a Person in AddressBook
+ * Guarantees: immutable
+ */
+public class Appointment implements Comparable<Appointment> {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Appointment constraints";
+
+    private static int idTracker = 1;
+    public final int appointmentId;
+    public final LocalDateTime appointmentDate;
+    public final int studentId;
+    public final String appointmentDescription;
```

Comment:
I don't think appointmentDescription is actually part of the feature specs? Or perhaps you're thinking of combining appointments with case logs?

I wouldn't be opposed to either, since case logs are essentially one-to-one with appointments and only adds one field: description

Reply:
this is a temporary field
because we might need a description for caselog column in v1.2 gui anyways